Player Script added to Sprite

Iam-B0B opened the pull request:
Made Prefabs for the character and the projectile.
Both have scripts added to them and animations made. 
they are not on the right layer and need to be moved up on top of  the background 
Both items have also been tagged "Player" and "PlayerProjectile" respectively

Iam-B0B committed:
Player Script added to Sprite

Death animation made as well

Iam-B0B committed:
Tags and death anim

tags and death anim for player

MV1090 reviewed (commented):
Everything seems to be working fine, I opened it up and played around with the player, and the overall functionality is good.
There are a couple of tweaks I would like to see made, if you can set gravity to 0 in the projectile rigidbody so it keeps heading up on the y axis, 'die' is missing from the transition between animations, and like you already said the player needs to be moved up in the layers.

Iam-B0B committed:
new folder for animation

Gravity 0 on projectile and 'die' has been added to the transition

Iam-B0B committed:
Allows for player to be spawned

Ziyad's requested chnages have been made as the player now has a spawnpoint and spawns in as a prefab

Iam-B0B committed:
correct changes

made a mistake and corrected them

Iam-B0B committed:
masses altered for player

chnaged to 0

Iam-B0B committed:
Merge branch 'master' into Player-Prefab-and-animation

# Conflicts:
#	Assets/Scenes/Game.unity
#	ProjectSettings/TagManager.asset
